docs/dns/dns-metrics.md

```diff
@@ -1,5 +1,5 @@
 <!--- Hugo front matter used to generate the website version of this page:
-linkTitle: DNS
+linkTitle: Metrics
```

Comment:
I'm not entirely sure now, but IIRC the website uses this as the text for the left-side menu navigation. This definitely cannot be just "metrics". Same this for the CLI programs -> Spans. 

Have you checked how these changes will affect the website?